assets/prototype/application/services/apollo/mutations/useEntityMutation.ts

```diff
@@ -118,10 +121,24 @@ const useEntityMutation = (type: EntityType, id?: string): EntityMutation => {
 		return { mutation, ...options };
 	};
 
+	/**
+	 * @param {string} mutationType Type of mutation - CREATE|UPDATE|DELETE
+	 * @param {string} suffix       Suffix to be added to mutation type
+	 */
+	const getToasterMessage = (mutationType: MutationType, suffix: 'ing' | 'ed' = 'ing'): any => {
```

Comment:
Maybe we can pull out these types?
the `suffix: 'ing' | 'ed' = 'ing'` part was a bit unclear at first :)